docs/future-work/motor-system-improvements/theory-for-decomposing-goals-into-subgoals.md

```diff
@@ -1,8 +1,17 @@
 ---
-title: Decompose Goals into Subgoals & Communicate
+title: Theory for Decomposing Goals into Subgoals
```

Comment:
Generally we haven't put theoretical work into the Future Work sections, I wonder whether it's best to reframe this, or what do you think? E.g. we could just remove it from Future Work for now, in the same way that we didn't really have behavior modelling in Future Work until we started getting more concrete ideas.

Reply:
Ah ok I had forgotten that we had that there. Happy to keep it then.